src/main/java/pl/kinga/task/TaskController.java

```diff
@@ -0,0 +1,77 @@
+package pl.kinga.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+@RequestMapping
+public class TaskController {
+
+    private TaskRepository taskRepository;
+    private List<Task> tasks;
+
+    @Autowired
+    public TaskController(TaskRepository taskRepository) {
+
+        this.taskRepository = taskRepository;
+    }
+
+    @GetMapping("/")
+    public String home() {
+        return "home";
+    }
+
+
+    @GetMapping("/toDo")
+    public String toDo(Model model) {
+        List<Task> allTasks = taskRepository.findByDoneTrue();
+        model.addAttribute("allTasks", allTasks);
+        return "toDo";
+    }
+
+    @GetMapping("/done")
+    public String done(Model model) {
+        List<Task> doneTasks = taskRepository.findByDoneFalse();
```

Comment:
jak jest możliwość, to robimy jak najbardziej generyczne metody, dlatego w tym przypadku zdecydowanie lepiej byłoby zrobić findByDone(false), wtedy będziesz mogła wykorzystać tą samą metodę, żeby znaleźć skończone zadania